lib/raidenclient/RaidenClient.ts

```diff
@@ -203,6 +235,57 @@ class RaidenClient extends SwapClient {
     // not implemented, raiden does not use invoices
   }
 
+  public lookupPayment = async (rHash: string, currency?: string, destination?: string) => {
```

Comment:
Same comment as for lnd's `lookupPayment` -> I think we should consider integration testing this in addition to upcoming simulation tests.